app/src/main/java/org/simple/clinic/drugs/selection/custom/CustomDrugEntryEffect.kt

```diff
@@ -43,3 +43,5 @@ data class RemoveDrugFromPrescription(val drugUuid: UUID) : CustomDrugEntryEffec
 object LoadDrugFrequencyChoiceItems : CustomDrugEntryEffect()
 
 object HideKeyboard : CustomDrugEntryEffect()
+
```

Comment:
Shouldn't the keyboard changes be in a different PR, considering this PR is for adding progress state in the buttons?

Reply:
Raising a different pr for it 👍🏻